cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -96,7 +101,12 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 	}
 
 	stopCh := make(chan struct{})
+	// This gives the post start hooks some time to run even if the actual
+	// tests run too fast. Otherwise post start hooks like
+	// crd-informer-synced may panic due to timeout.
+	minRunTime := time.After(10 * time.Second)
```

Comment:
I think we should only do this if the `StorageVersionWrapFunc` is set.  Otherwise this shouldn't be needed, right?

Reply:
actually I'm not sure why we need this. The description isn't related to the poststart hook being tested here. If a test calling `teardown` right after it sees `/healthz` returning `ok` would cause some other poststart hook to panic/flake, it's better to fix those hooks. Or we may just make the specific tests with such behavior to wait, instead of adding latency to all test. 

Removed this to see what happens.

Reply:
Talked with @caesarxuchao offline. Likely this is not needed anymore (fixed somewhere else)